models/balancer/ethereum/balancer_ethereum_sources.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+
+sources:
+  - name: balancer_ethereum
+    freshness:
```

Comment:
it appears this source shouldn't apply a freshness check:
```
select
      max(evt_block_time) as max_loaded_at,
      current_timestamp() as snapshotted_at
    from balancer_ethereum.BFactory_evt_LOG_NEW_POOL
```

Reply:
Fixed.

Removed the DBT freshness check.